TWLight/applications/views.py

```diff
@@ -480,6 +484,7 @@ def get_success_url(self):
                     )
                 )
             ),
```

Comment:
closing brackets for `_()` should be lifted to the same line as the source string